www/source/blog/2017-11-30-Windows-Docker-Export.html.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Exporting Windows Packages to Docker"
+date: 2017-11-30
+author: Matt Wrock
+tags: Windows, Supervisor
+category: Windows
+classes: body-article
+---
+
+Today's Habitat 0.50.0 release introduces the ability to export a Windows package to a Docker Windows container. This is great for multiple reasons! For starters:
+
+* You can easily test multi supervisor configurations without provisioning VMs or fiddling with alternate supervisor ports.
+* Habitat becomes a great onboarding path to containerizing your Windows applications.
+* The containerization gives you an added level of isolation and repeatability - Currently you can assume a vanilla Windows 2016 Server Core base image and don't need to guess what dependencies or other services may have snuck on to that instance.
+
+We use `microsoft/windowsservercore` as our base image. So if your package needs features or other windows settings not included in that default image, make sure your `init` hook installs them.
+
+Lets walk through containerizing a "Habetized" ASP.NET Core application that accesses a MySQL database. We will create two images. One for the web application and the other for the database. Then we will run both and get them talking to eachother.
```

Comment:
"Habitized", at least, though it'd be nice for us to formally figure out what the verb form of "Habitat" is 😄

Reply:
"two images: one for the web application..."

Reply:
"each other"

Reply:
s/lets/Let's